EIPS/eip-6811.md

```diff
@@ -0,0 +1,49 @@
+---
+eip: 6811
+title: Ethereum To The Moon!
+description: Makes a minimal number of changes that allow Ethereum to be used on the moon
+author: Pandapip1 (@Pandapip1)
+discussions-to: https://ethereum-magicians.org/t/ethereum-to-the-moon/13633
+status: Draft
+type: Standards Track
+category: Core
+created: 2023-04-01
+---
+
+## Abstract
+
+This EIP makes a minimal number of changes to allow Ethereum to be used on the moon and other potentially habitable bodies in Earth's solar system. It changes the time between blocks, the per-block validator reward, and the number of blocks per epoch.
+
+## Motivation
+
+It is impossible for Ethereum to literally "go to the moon" due to a limitation in the protocol: the block length. Should validators attempt to validate on the surface of the moon, they would find that the ~1.25 second communication delay (caused by the speed of light) might cause issues with synchronization, considering the 12-second timer between blocks. The validators would find themselves slashed on the terrestrial chain, and validating their own fork.
```

Comment:
validators won't be slashed per se but they would `leak` and be force exited in each other's forks
```suggestion
It is impossible for Ethereum to literally "go to the moon" due to a limitation in the protocol: the block length. Should validators attempt to validate on the surface of the moon, they would find that the ~1.25 second communication delay (caused by the speed of light) might cause issues with synchronization, considering the 12-second timer between blocks. The validators would eventually be ejected on the terrestrial chain after leaking. If however a substantial number of validators are displaced (think 1/3), they might follow their own fork and would eventually eject the terrestrial to finalize their own chain.
```